### PR TITLE
[TEST] DO NOT MERGE OR REVIEW UNLESS ASKED

### PR DESCRIPTION
**Pull request by lock14**
Testing the behavior of having a non-owner approve a PR

### CODE REVIEW

**Review by crwilcox — APPROVED** [review]

**Comment by lock14** [comment]
The test for this PR was successful. Closing it out. If anyone was interested what the test was. We wanted to know what impact a non-owner (as determined by a repository's CODEOWNERS file ) approving a PR had on the `reviewDecision` field of GitHub's GraphQL API. The result was the owner is still required to approve before the PR is marked as 'APPROVED' in the API.

https://screenshot.googleplex.com/4aZdb2THeWcrUrb

**Comment by sethvargo** [comment]
@lock14 am I able to remove the GraphQL authorization now? Or do you still need it?

**Comment by lock14** [comment]
@sethvargo yes you can go ahead and remove it. I don't need access anymore.

**Pull request closed without merging**
